Microsoft/edx-configuration
# correcting the typo

What's this PR do? Any additional context?

This will add the values of EMAIL in setting files. I have corrected the typo.

Where should the reviewer start?

he should deploy the fix and check the key in lms.env.json

How can this be manually tested? (brief repro steps)

This can be tested by checking the added keys in lms.env.json after deployment

What are the relevant TFS items? (list id numbers)

98998

Definition of done:

 Title of the pull request is clear and informative
 Add pull request hyperlink to relevant TFS items
 For large or complex change: schedule an in-person review session
Reminders BEFORE merging

Get at least two approvals
If you're merging into the development branch then "flatten" or "squash" commits
If merging from development into master then don't "flatten" or "squash" commits
Reminders AFTER merging

Delete the remote branch
Resolve relevant TFS items
(reverse merge) If you merged into master then check to see if there are any changes in master that can be merged down to the development branch (like hotfixes, etc)
Configuration Pull Request

(For changes proposed to upstream)

Make sure that the following steps are done before merging

 @devops team member has commented with 👍
 are you adding any new default values that need to be overridden when this goes live?
 Open a ticket (DEVOPS) to make sure that they have been added to secure vars.
 Add an entry to the CHANGELOG.
